### PR TITLE
Add log submission test

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
#2341 

I removed `UITestInfoActivity` as it's only purpose was to support the calabash tests, and since now we've moved away from it, there is no point in having the activity anymore.

### CODE REVIEW

File: app/instrumentation-tests/src/org/commcare/utils/InstrumentationUtility.kt [hunk 1]
```diff
@@ -325,6 +327,11 @@ object InstrumentationUtility {
                 .perform(click())
     }
 
+    fun loadLogSubmissionInfo(): Boolean {
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/utils/InstrumentationUtility.kt:330]: nit: rename to "getLastLogSubmissionResult` or `didLastLogSubmissionSucceed`

**Review by shubham1g5 — APPROVED** [review]

**Comment by ShivamPokhriyal** [comment]
@damagatchi retest this please

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462096/)

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462098/)

**Review by shubham1g5 — APPROVED** [review]

**Pull request merged**